### PR TITLE
Mark python3-py as unwanted in RHEL 10+

### PR DESCRIPTION
**Pull request by befeleme**

### CODE REVIEW

File: configs/sst_cs_apps-unwanted-python3-eln.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ data:
   # tomllib is in the Python standard library (PEP 680), use that one instead
   - python3-toml
   - python3-tomli
+  # python3-py was needed by pytest which bundled the necessary parts
```
Comment (hroncok) [inline configs/sst_cs_apps-unwanted-python3-eln.yaml:19]: What about this?

```suggestion
  # python3-py is in maintenance mode upstream, pretty much dead
  # it used to be in RHEL as a dependency of pytest which bundled the necessary parts
```

File: configs/sst_cs_apps-unwanted-python3-eln.yaml [hunk 2]
```diff
@@ -27,6 +29,7 @@ data:
   unwanted_source_packages:
   - pytz
   - python-chardet
+  - python-py
   - python-toml
   - python-tomli
```
Comment (hroncok) [inline configs/sst_cs_apps-unwanted-python3-eln.yaml:34]: Perhaps keep the same order here?

```suggestion
  - python-toml
  - python-tomli
  - python-py
```

**Review by hroncok — COMMENTED** [review]
Ack for the actual content change. 2 nitpicks.

**Comment by befeleme** [comment]
Thank you. I accepted both changes.

**Review by hroncok — APPROVED** [review]
Thanks!

**Pull request merged**